Extensions/PlatformBehavior/JsExtension.cpp

```diff
@@ -104,8 +104,23 @@ class PlatformBehaviorJsExtension : public gd::PlatformExtension {
           .SetFunctionName("setJumpSustainTime")
           .SetGetter("getJumpSustainTime");
       autExpressions["JumpSustainTime"].SetFunctionName("getJumpSustainTime");
-
+      autConditions["PlatformBehavior::CurrentFallSpeed"].SetFunctionName(
+          "getCurrentFallSpeed");
+      autExpressions["CurrentFallSpeed"].SetFunctionName("getCurrentFallSpeed");
+      autConditions["PlatformBehavior::CurrentSpeed"].SetFunctionName(
+          "getCurrentSpeed");
+      autExpressions["CurrentSpeed"].SetFunctionName("getCurrentSpeed");
+      autActions["PlatformBehavior::CanGrabPlatforms"]
+          .SetFunctionName("setCanGrabPlatforms")
+          .SetGetter("getCanGrabPlatforms");
+      autConditions["PlatformBehavior::CanGrabPlatforms"].SetFunctionName(
+          "getCanGrabPlatforms");    
```

Comment:
nit: let's call this function simply `canGrabPlatforms` - the get seems redundant with the "can". Rule of thumb, functions returning a boolean should be called:`is[State]()`, `can[Ability]()` or `has[Something]()`.
Same for the C++ function.